release-notes-3.7.0.md

```diff
@@ -1,8 +1,17 @@
-!not-ready-for-release!
 
 #### Version Number
 ${version-number}
 
 #### New Features
+- **SCMOD-9181**: Updated jre11 to support generic SSL cert dir. The "MESOS_SANDBOX" variable has been replaced with "SSL_CA_CRT_DIR" in order to maintain backward compatibility (ex: for use on Kubernetes)
+
+- **SCMOD-5156**: Fully-qualified DockerHub images. DockerHub security: Abandon the generic hpeemployee account as we have the internal DockerHub proxy on dockerhub-private.svsartifactory.swinfra.net.
+  
+- **SCMOD-11069**: Updated to use the latest release of base image (opensuse-jre11-3.1.0-SNAPSHOT). Release the whole hierarchy of base images and update the services that are currently using pre-release versions of the base images to use the new released versions of the base images.
```

Comment:
We shouldn't be referencing SNAPSHOT images in release notes (and it's not SNAPSHOT anymore in master).

I don't think we need to call out SCMOD-11069 in the release notes as it was just for releasing the base images. We should just call out SCMOD-9182 instead I think.

Also, use the same formatting as that in the example release notes (but don't use hyperlinks):

https://github.com/WorkerFramework/worker-framework/releases/tag/v1.6.0

For example, start - space - title - 2 spaces
2 spaces - descripton

Here is what I suggest these release notes should look like:

* SCMOD-9181: Updated JRE 11 base image  
    The JRE 11 base image has been updated to support a generic SSL cert directory. The `MESOS_SANDBOX` variable has been replaced with `SSL_CA_CRT_DIR` in order to maintain backward compatibility (ex: for use on Kubernetes).
    
* SCMOD-5156: Use fully-qualified DockerHub images  
    DockerHub image names have been updated so that images are pulled from our DockerHub proxy.
    
* SCMOD-11768: Explicitly specify base image
    Explicitly specify the busybox base image so that it is pulled from our DockerHub rather than `docker.io`.